_analyzers/character-filters/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Character filters
+nav_order: 90
+has_children: true
+has_toc: false
+---
+
+# Character filters
+
+Character filters process the text before tokenization, modifying, or cleaning the input to prepare it for further analysis.
+
+Unlike token filters, which operate on tokens (words or terms), character filters work on the raw input text before tokenization. They are especially useful for cleaning or transforming structured text with unwanted characters, such as HTML tags or special symbols. Character filters help strip or replace these elements so that text is properly formatted for analysis.
```

Comment:
```suggestion
Unlike token filters, which operate on tokens (words or terms), character filters process the raw input text before tokenization. They are especially useful for cleaning or transforming structured text containing unwanted characters, such as HTML tags or special symbols. Character filters help to strip or replace these elements so that text is properly formatted for analysis.
```